roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -2,6 +2,7 @@
 # We should print out deprecations prior to any failures so that if a play does fail for other reasons
 # the user would also be aware of any deprecated variables they should note to adjust
 - include_tasks: deprecations.yml
+  when: not openshift_skip_deprecation_check | default(false) | bool
```

Comment:
Put parenthesis around 'openshift_skip_deprecation_check | default(false)' for clarity.